model_zoo/models/alexnet/model_alexnet.prototext

```diff
@@ -267,15 +301,28 @@ model {
 
   layer {
     name: "prob"
+    parents: "fc8"
     data_layout: "model_parallel"
     softmax {}
   }
 
+  layer {
+    name: "cross_entropy"
+    parents: "prob labels"
+    data_layout: "model_parallel"
+    cross_entropy {}    
+  }
+  layer {
+    name: "cross_entropy_eval"
+    parents: "cross_entropy"
+    data_layout: "model_parallel"
+    evaluation {}    
+  }
   layer {
     name: "target"
-    parents: "prob data"
+    parents: "prob labels"
     data_layout: "data_parallel"
     target {}
-  }
+  }  
```

Comment:
What is the role of the target layer with the cross entropy layers.

Reply:
Accuracy is still computed with the target layer. If we don't need to report accuracy, the target layer can be removed.